modular_zubbers/code/modules/surgery/organs/internal/lungs/_brain.dm

```diff
@@ -0,0 +1,2 @@
+/obj/item/organ/internal/brain
+	resistance_flags = INDESTRUCTIBLE
```

Comment:
this overrides the ones defined originally- are any defined? Not sure if it should be added in New()... or not.

Reply:
This flag is above all the others, in my testing this includes lava and acid resistance. And there are none